universal-application-tool-0.0.1/app/services/question/types/QuestionDefinition.java

```diff
@@ -240,6 +242,15 @@ public String getValidationPredicatesAsString() {
           .anyMatch(option -> option.optionText().hasEmptyTranslation())) {
         errors.add(CiviFormError.of("Multi-option questions cannot have blank options"));
       }
+      int numUniqueOptionDefaultValues =
+          multiOptionQuestionDefinition.getOptions().stream()
+              .map(QuestionOption::optionText)
+              .map(LocalizedStrings::getDefault)
+              .collect(Collectors.toSet())
```

Comment:
thought (non-blocking): I think you can also use `.distinct().count()`, but collecting to a set works well too

Reply:
oo i like that